phpunit.xml

```diff
@@ -8,7 +8,6 @@
          convertWarningsToExceptions="true"
          processIsolation="false"
          stopOnFailure="false"
-         syntaxCheck="false"
```

Comment:
Why this removal?

Reply:
There's a configuration warning on PHPUnit 7, not sure about 6.

```sh
Michaels-MacBook-Pro:slack mbabker$ vendor/bin/phpunit 
PHPUnit 7.5.2 by Sebastian Bergmann and contributors.

  Warning - The configuration file did not pass validation!
  The following problems have been detected:

  Line 12:
  - Element 'phpunit', attribute 'syntaxCheck': The attribute 'syntaxCheck' is not allowed.
```